crates/solver/src/driver.rs

```diff
@@ -332,7 +335,10 @@ impl Driver {
                         })
                         .collect(),
                     call_data: settlement_simulation::call_data(
-                        rated_settlement.settlement.clone().into(),
+                        rated_settlement
+                            .settlement
+                            .clone()
+                            .into_encoded(InternalizationStrategy::SkipInternalizableInteraction), // rating is done with internalizations
```

Comment:
naming-nit: I would just call this method "encode".

Reply:
Renamed.